### ISSUE TITLE
ScraperException Error

### ISSUE THREAD

**Issue by oroszkinorbert** [JustAnotherArchivist/snscrape#764]
### Describe the bug

ScraperException: 4 requests to https://api.twitter.com/2/search/adaptive.json?include_profile_interstitial_type=1&include_blocking=1&include_blocked_by=1&include_followed_by=1&include_want_retweets=1&include_mute_edge=1&include_can_dm=1&include_can_media_tag=1&include_ext_has_nft_avatar=1&include_ext_is_blue_verified=1&include_ext_verified_type=1&skip_status=1&cards_platform=Web-12&include_cards=1&include_ext_alt_text=true&include_ext_limited_action_results=false&include_quote_count=true&include_reply_count=1&tweet_mode=extended&include_ext_collab_control=true&include_ext_views=true&include_entities=true&include_user_entities=true&include_ext_media_color=true&include_ext_media_availability=true&include_ext_sensitive_media_warning=true&include_ext_trusted_friends_metadata=true&send_error_codes=true&simple_quoted_tweet=true&q=NuclearEnergy+min_faves%3A50+since%3A2021-10-01+until%3A2023-03-10%2C+top+%3DTrue+&tweet_search_mode=live&count=20&query_source=spelling_expansion_revert_click&pc=1&spelling_corrections=1&include_ext_edit_control=true&ext=mediaStats%2ChighlightedLabel%2ChasNftAvatar%2CvoiceInfo%2Cenrichments%2CsuperFollowMetadata%2CunmentionInfo%2CeditControl%2Ccollab_control%2Cvibe failed, giving up.

### How to reproduce
```python3
import xlsxwriter
import snscrape.modules.twitter as sntwitter
import pandas as pd

query ="NuclearEnergy"
limit=10000

tweets=[]
for tweet in sntwitter.TwitterSearchScraper('NuclearEnergy min_faves:50 since:2021-10-01 until:2023-03-10, top =True ').get_items():
    if len(tweets) == limit:
        break
    else:
        tweets.append([tweet.viewCount, tweet.date,  tweet.url, tweet.user.username, tweet.content, tweet.likeCount, tweet.retweetCount, tweet.replyCount, tweet.quoteCount])
        
df = pd.DataFrame(tweets, columns=['Views','Date','Url', 'User', 'Tweet', 'Like', 'Retweet', 'Reply', 'Quote'])
date_columns = df.select_dtypes(include=['datetime64[ns, UTC]']).columns
for date_column in date_columns:
    df[date_column] = df[date_column].dt.date


workbook =xlsxwriter.Workbook('C:/Users/Norbi/Desktop/NuclearEnergy.xlsx')
df.to_excel('C:/Users/Norbi/Desktop/NuclearEnergy.xlsx',"Munka1")
```

### Expected behaviour

Few days ago I got all the tweets in my excel file, but now I got this issue.

### Operating system

Win 11

### Python version: output of `python3 --version`

Python 3.9

### snscrape version: output of `snscrape --version`

 snscrape-0.6.1.20230314

### Scraper

TwitterSearchScraper

### How are you using snscrape?

Module (`import snscrape.modules.something` in Python code)

**Comment by kareemrasheed89** [comment]
Got the same error this morning and am currently using the dev version of the code.

**Comment by om-varshney** [comment]
I am getting the same error. It was working well until yesterday evening. I am using snscrape version 0.5.0

**Comment by JustAnotherArchivist** [comment]
The syntax on the `TwitterSearchScraper` argument seems wrong, but that would just return no results, not trigger an error.

Please enable logging at the debug level (`import logging; logging.basicConfig(level = logging.DEBUG)`).

**Comment by rayenebech** [comment]
I was getting the same error, till I updated the user agent with the new commit ([see here](https://github.com/JustAnotherArchivist/snscrape/commit/e47fbe3d1f078582d24d4b7b04f0ab4e41ddff1c)) and it worked fine :)

**Comment by JustAnotherArchivist** [comment]
@om-varshney @rayenebech That is fixed in snscrape 0.6.1. This issue is different, since @oroszkinorbert and @kareemrasheed89 claim they still see it on that version.

**Comment by kareemrasheed89** [comment]
> @om-varshney @rayenebech That is fixed in snscrape 0.6.1. This issue is different, since @oroszkinorbert and @kareemrasheed89 claim they still see it on that version.

Thanks @JustAnotherArchivist what can one do to escape this error, I just got that this morning.

**Comment by JustAnotherArchivist** [comment]
@kareemrasheed89 Use the latest snscrape version, or provide a debug log as requested above if you still get it with 0.6.1.
